docs/how-to/multicluster-gateways-walkthrough.md

```diff
@@ -209,88 +258,9 @@ So what about DNS how do we bring traffic to these gateways?
     EOF
     ```
 
-### Enable DNS
-
-1. Create a DNSPolicy and attach it to your Gateway:
-
+6. Let's create the same HTTPRoute in the second workload cluster. Note the `--context` references the second cluster
     ```bash
-    kubectl --context kind-mgc-control-plane apply -f - <<EOF
-    apiVersion: kuadrant.io/v1alpha1
-    kind: DNSPolicy
-    metadata:
-      name: prod-web
-      namespace: multi-cluster-gateways
-    spec:
-      targetRef:
-        name: prod-web
-        group: gateway.networking.k8s.io
-        kind: Gateway     
-    EOF
-    ```
-
-   Once this is done, the Kuadrant multi-cluster gateway controller will pick up that a HTTPRoute has been attached to the Gateway it is managing from the hub and it will setup a DNS record to start bringing traffic to that gateway for the host defined in that listener.
-
-2. You should now see a DNSRecord resource in the hub cluster:
-
-    ```bash
-    kubectl --context kind-mgc-control-plane get dnsrecord -A
-    ```
-    ```
-    NAMESPACE                NAME                 READY
-    multi-cluster-gateways   prod-web-api         True
-    ```
-
-3. You should also be able to see there is only 1 endpoint added which corresponds to the address assigned to the gateway where the HTTPRoute was created:
-
-    ```bash
-    kubectl --context kind-mgc-control-plane get dnsrecord -n multi-cluster-gateways -o=yaml
-    ```
-
-4. Give DNS a minute or two to update. You should then be able to execute the following and get back the correct A record. 
-
-    ```bash
-    dig echo.$MGC_ZONE_ROOT_DOMAIN
-    ```
-5. You should also be able to curl that endpoint
-
-    ```bash
-    curl -k https://echo.$MGC_ZONE_ROOT_DOMAIN
-
-    # Request served by echo-XXX-XXX
-    ```
-
-## Introducing the second cluster
-
-So now we have a working gateway with DNS and TLS configured. Let place this gateway on a second cluster and bring traffic to that gateway also.
-
-1. First add the second cluster to the clusterset, by running the following:
-
-    ```bash
-    kubectl --context kind-mgc-control-plane label managedcluster kind-mgc-workload-1 ingress-cluster=true
-    ```
```

Comment:
This command was removed from the walkthrough because it was added as part of the deployment in the workload clusters, so there's no need to run it manually